src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -188,6 +188,16 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
         auto thickness = _ParseThicknessFromPadding(_settings.Padding());
         _swapChainPanel.Margin(thickness);
 
+        if (thickness.Bottom != 0 || thickness.Top != 0 || thickness.Left != 0 || thickness.Right != 0)

```

Comment:
This is going to fire everytime someone's settings reloads, even if the padding didn't change, right? Should we be doing any filtering of that on our end?

Reply:
Alas, it will. It may be a little noisy, but I don't necessarily think it's worth complicating the logic to filter it out.